### PR TITLE
font-awesome

### PR DESCRIPTION
**Pull request by robertdown**
Add font-awesome in a new folder (interface/static) which will hold all our global CSS/JS files and change the icons on the left menu

### CODE REVIEW

**Comment by bradymiller** [comment]
Hi Robert,
This looks very nice. Recommend posting this on the forum to ensure community is ok with the path where storing the font-awesome library.
-brady

**Comment by olegsv** [comment]
@robertdown nice work.
Why do you think we need to bring SCSS source files under the same folder?

**Comment by robertdown** [comment]
@olegsv The font-awesome folder is the full font-awesome download, installed via the Bower package manager. It contains everything they provided, I did not add, remove, or change anything. We *could* strip out everything but the final build, but since we aren't making any changes to the library I think we should just leave everything the way it is.

@bradymiller this was already [discussed](https://sourceforge.net/p/openemr/discussion/202506/thread/53f41e90/#5ec5/ef11/166d/06a3/9ecb/6d5d) in the forums. `interface/static` will represent external files/libraries such as bootstrap and font-awesome.

Note I added a .bowerrc and bower.json file that helps to manage this

**Comment by robertdown** [comment]
Link doesn't seem to be working... it's on that thread, page 4, about halfway down

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "openemr",
+  "homepage": "http://open-emr.dev",
```
Comment (bradymiller) [inline bower.json]: should this be open-emr.org ?

Comment (robertdown) [inline bower.json]: No, that can be just OpenEMR. I believe that's more for if we were planning on publishing a package but when I ran 'bower init' it asked for a name so I put one in

**Comment by robertdown** [comment]
Note I just rebased the latest changes from master onto this branch and did a force push. Still no conflicts :)

**Comment by bradymiller** [comment]
Hi Robert,

I just rebased these commits into 1 commit and brought it into the main codebase.

I also created a bower wiki page here to track things and not "lose" this feature: http://www.open-emr.org/wiki/index.php/OpenEMR_Wiki_Home_Page#Embedded_Components

I am guessing bootstrap will be next :)

-brady

**Pull request closed without merging**